posthog/temporal/data_imports/external_data_job.py

```diff
@@ -62,11 +63,14 @@ async def create_external_data_job_model(inputs: CreateExternalDataJobInputs) ->
         database = source.job_inputs.get("database")
         schema = source.job_inputs.get("schema")
         schemas_to_sync = await sync_to_async(get_postgres_schemas)(host, port, database, user, password, schema)
-        await sync_to_async(sync_old_schemas_with_new_schemas)(  # type: ignore
-            schemas_to_sync,
-            source_id=inputs.external_data_source_id,
-            team_id=inputs.team_id,
-        )
+    else:
+        schemas_to_sync = list(PIPELINE_TYPE_SCHEMA_DEFAULT_MAPPING.get(source.source_type, ()))
```

Comment:
Reminder: i think we'll still be broken here because of permission differences that we mentioned with hubspot integration

Reply:
Ah, I'm unaware of any hubspot permissions issue. Lets chat at standup

Reply:
@EDsCODE I actually think this is okay - It's the lesser of 2 evils right now. Something I'd like to fix-forward with statuses for each table (some failed due to mismatch permissions, etc) - shall we get this in and move on?